sdk/va-report-components/documentation/docs/api/ObjectHandle.md

```diff
@@ -39,6 +39,23 @@ If no `options` parameter is supplied, the report is exported using the default
 Refreshes the data for the report object that is controlled by the
 `ObjectHandle`.
 
+### getData(options?: Object): ReportObjectResultData[]
+
+Returns all currently displayed data from the report object.
```

Comment:
"currently displayed" has an odd ring to me.  Is it not enough to just say "Returns all data from the report object."?  I am contrasting this with what is already documented for getSelectedData, which is "Returns a user's selection data from the report object."

Reply:
@brmorr, you make a good point. Using "Returns all data from the report object." is clearer.

Reply:
@brmorr @joylashley I added the 'currently displayed' bit to subtly indicate that if some of the object's data is not displayed it wouldn't return from the api (ie. if there is a filter applied that filters out part of the data). Is that an edge case we should specify more outright, or leave off entirely?

Reply:
I assumed that was the subtly you were handling with that wording.  But thinking about it a bit more... an "object's data" is exactly what is visualized.  If there is a filter, then I see that filtered out portion as no longer part of the objects data.  I think we are probably fine to just leave the simpler wording.  But if we feel strongly about providing some nuanced explanation then I would think it might be handled in a 2nd sentence, following the more generic initial description sentence.  So something like

"Returns all data from the report object.  This data matches what the report object is displaying, including any summarization or filtering that has been applied."

Reply:
@eheckel and @brmorr, adding further explanation would be helpful. Brad, would it be sufficient to mention just filters in the second sentence? What else are you indicating by "summarization"? I haven't typically used the term 'summary' in VA doc unless it's referring to a report summary, which is just text.

I have a small tweak for the first sentence that I missed earlier and a possibility if only filters are mentioned in the second sentence. "Returns all of the data from the report object. This data matches what appears in the report object, including any filters that have been applied."

Reply:
Thank you both; I've applied the suggested wording from Joy's most recent comment.

Reply:
@joylashley , the word I was really thinking was "aggregation".  But that is really getting into the details of what is happening to the data on its way from raw data table to the query results specific for that object.  I'm fine with it as is, just using filters.